src/app/payments/helpers.ts

```diff
@@ -159,6 +161,37 @@ export const newCheckIntraledgerLimits = async ({
   })
 }
 
+export const newCheckTradeIntraAccountLimits = async ({
+  amount,
+  wallet,
+  priceRatio,
+}: {
+  amount: UsdPaymentAmount
+  wallet: Wallet
+  priceRatio: PriceRatio
+}) => {
+  const timestamp1Day = new Date(Date.now() - MS_PER_DAY)
```

Comment:
there is 11 instances of this line in our codebase. maybe a good time to rebase in an util function.

would be handy the day we need weekly or monthly limit.

Reply:
do we ever pass anything other than 1 day to `tradeIntraAccountTxBaseVolumeAmountSince`? Because perhaps we could just delete the line and call the fn:
```
tradeIntraAccountTxBaseVolumeAmountInLast24Hours
```

Reply:
> do we ever pass anything other than 1 day to `tradeIntraAccountTxBaseVolumeAmountSince`? Because perhaps we could just delete the line and call the fn:
> 
> ```
> tradeIntraAccountTxBaseVolumeAmountInLast24Hours
> ```

We could do this. I'd favour doing it in a follow-up PR tho since it would touch a lot of functions

Reply:
1 month will be another typical need. Potentially 1 week ago. So o would not hardcore the duration for now